bin/compile.js

```diff
@@ -29,6 +30,7 @@ const options = {
   target: program.target,
   bytecode: !!program.bytecode,
   babelify: program.babelify,
+  sourcemap: !!program.sourcemap,
```

Comment:
Is this correct after the switch got renamed?

Reply:
yes adding --no prefix inverts it via commander.

Reply:
Ahh right! I guess we don't need the `!!` to ensure a boolean here btw? (Seeing as we don't do the conversion for `babelify`.)

Reply:
Updated(removed !!)